core/db_models/EEM_Form_Section.model.php

```diff
@@ -98,22 +98,22 @@ protected function __construct(FormStatus $form_status, ?string $timezone)
                         'JSON string of HTML attributes, such as class, to be applied to this form section\'s container.',
                         'event_espresso'
                     ),
-                    true,
+                    false,
                     '{}'
```

Comment:
why deal with null cases when you can set an appropriate default?
(i mean other than the obvious answer that the db runs more efficiently with null values)

Reply:
Yeah, that's better. I wanted to the same the other day, but thought that there might be some underlying logic that uses those values.